assets/scripts/gallery/fetch_street.js

```diff
@@ -44,6 +44,7 @@ export function fetchGalleryStreet (streetId) {
 
 function errorReceiveGalleryStreet (err) {
   console.log(err)
+  showError(ERRORS.STREET_FETCH_FAILURE, false)
```

Comment:
@flickz You missed one name change, here

Reply:
Oops! Really sorry for that